cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraJobInfo.java

```diff
@@ -50,6 +50,10 @@ public ConsistencyLevel getConsistencyLevel()
     @Override
     public String getLocalDC()
     {
+        if (isCoordinatedWriteEnabled())
+        {
+            throw new UnsupportedOperationException("Read localDc per cluster from coordinatedWriteConf, when coordinated write is enabled");
```

Comment:
I'm not understanding the message here.
```suggestion
            throw new UnsupportedOperationException("Remove the 'LOCAL_DC' writer option when using coordinated writes");
```

Reply:
Yeah. The message is not suitable for project internal callsites. 

I have updated the message to "Method not supported, when coordinated write is enabled. Call-sites should get localDc from #coordinatedWriteConf() method"